kart/point_cloud/import_.py

```diff
@@ -0,0 +1,147 @@
+from glob import glob
+import json
+from os.path import expanduser
+import sys
+
+import click
+
+from kart.exceptions import (
+    InvalidOperation,
+    NotFound,
+    NO_IMPORT_SOURCE,
+    INVALID_FILE_FORMAT,
+)
+from kart.output_util import format_wkt_for_output
+
+
+@click.command("point-cloud-import", hidden=True)
+@click.pass_context
+@click.argument("sources", metavar="SOURCES", nargs=-1, required=True)
+def point_cloud_import(ctx, sources):
+    """
+    Experimental command for importing point cloud datasets. Work-in-progress.
+    Will eventually be merged with the main `import` command.
+
+    SOURCES should be one or more LAZ or LAS files (or wildcards that match multiple LAZ or LAS files).
```

Comment:
Not sure. I guess it would help if...
... the shell didn't do globbing for whatever reason, like maybe if the user added quotes to deal with special characters like spaces. But now I guess it means you can't specify files with stars in the name....
... if we could only accept a single arg, and not multiple args. Depends slightly on how this works when we merge it with the existing import command.
... makes testing slightly easier to be able to specify globs directly.

Reply:
YAGNI. shells are good at globs

Reply:
Done

Reply:
Shouldn't the type be `click.Path`? 

Eventually we'll hit a line-length limit (Linux ~128K, Windows 8K, macOS 256K) but I guess then we add `@sources.txt`/`--files-from=sources.txt`/etc.

Reply:
Or indeed with quoted globs which I think I may add back in again at some stage to make this a bit easier for users - no point doing `echo *.laz > sources`, `kart import @sources` when you could just write `kart import "*.laz"` and I could make it work. But yeah will need read-from-file support too

Reply:
Wait I think `echo *.laz` won't work either due to line length. But the point still stands.